src/packages/cli/package.json

```diff
@@ -87,6 +88,7 @@
     "eslint-plugin-prettier": "3.1.4",
     "execa": "4.0.3",
     "fast-deep-equal": "3.1.3",
+    "fs-jetpack": "^3.1.0",
```

Comment:
Convenient API for working with the disk.

Reply:
Nice! Didn't know that one